extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -285,22 +285,6 @@ export class AnalyticsPlugin extends BaseTelemetryPlugin implements IAppInsights
                 if (doc) {
                     pageView.refUri = pageView.refUri === undefined ? doc.referrer : pageView.refUri;
                 }
-
-                let perf = getPerformance();
-                // Access the performance timing object
-                const navigationEntries = (perf && perf.getEntriesByType && perf.getEntriesByType("navigation"));
-
-                // Edge Case the navigation Entries may return an empty array and the timeOrigin is not supported on IE
-                if (navigationEntries && navigationEntries[0] && !isUndefined(perf.timeOrigin)) {
-                    // Get the value of loadEventStart
-                    const loadEventStart = (navigationEntries[0] as PerformanceNavigationTiming).loadEventStart;
-                    pageView.startTime =  new Date(perf.timeOrigin + loadEventStart);
-                } else {
-                    // calculate the start time manually
-                    let duration = ((properties || pageView.properties || {}).duration || 0);
```

Comment:
Looking at all of the "usages" of the `sentPageViewInternal`, it looks like (once we remove this from here), that we should do this duration update in the `AngularPlugin.ts Line 602` (before the `_pageTracking` calls the `_self.sendPageViewInternal` so that the "time" is still shifted back by the `duration` otherwise the Portal will show the wrong "startTime"

Reply:
Maybe?? (testing required)
We can just keep something here like

```
if (isNullOrUndefined(pageView.startTime)) {
    // calculate the start time manually
    let duration = ((properties || pageView.properties || {}).duration || 0);
    if (duration) {
        pageView.startTime = new Date(new Date().getTime() - duration);
    }
}
```

So that if the previous code (like the line 602 above) doesn't specify any `startTime` then we keep "using" this default, this should also work for the subsequent page views, even with your code below for the `if (!firstPageViewSent){`